### DISCUSSION TITLE
Automatically getting files out of ~/arm/media

### DISCUSSION THREAD

**Discussion by itsjop** [automatic-ripping-machine/automatic-ripping-machine#1155] [Q&A]
Hi there, I've got ARM running just fine (movies aren't identifying at all, but that's something I can likely correct by hand), but I can't seem to get the finished files out of the /arm user folder for use elsewhere. Either I've misunderstood how to configure the -v parameters (eg `-v "/home/arm/music:/home/arm/music"` for getting the files out of the `arm` user directory, or there has to be a way of getting them to a more convenient location on my main user account. Thanks in advance for any suggestions or instructions.

**Comment by studog-github** [comment]
You're probably running into permissions issues. You'll want the `chown` command perhaps.
